test_scripts/WebEngine/WebEngineProjection/009_OnSCU_after_Show_main.lua

```diff
@@ -0,0 +1,73 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0273-webengine-projection-mode.md
+--
+-- Description: Check that SDL transfer `OnSystemCapabilitiesUpdated` notification from HMI to an App
+-- which sent `Show` RPC with `templateConfiguration` parameter for main window after `Show` response
+--
+-- Preconditions:
+-- 1) SDL and HMI are started
+-- 2) App is registered
+-- Steps:
+-- 1) App sends `Show` request with `templateConfiguration` for main window (windowID is not defined)
+-- SDL does:
+--  - proceed with `Show` request successfully
+--  - not send `OnSystemCapabilityUpdated` to App
+-- 2) HMI sends `OnSystemCapabilityUpdated` to SDL
+-- SDL does:
+--  - transfer `OnSystemCapabilityUpdated` notification to App
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local common = require('test_scripts/WebEngine/commonWebEngine')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+local appHMIType = "WEB_VIEW"
+
+--[[ General configuration parameters ]]
+config.application1.registerAppInterfaceParams.appHMIType = { appHMIType }
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 7
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
+config.defaultMobileAdapterType = "WS"
+common.testSettings.restrictions.sdlBuildOptions = {{ webSocketServerSupport = { "ON" }}}
+
+--[[ Local Functions ]]
+local function sendShow()
+  local showParams = {
+    mainField1 = "MainField1",
+    templateConfiguration = {
+      template = "Template1"
```

Comment:
@LitvinenkoIra I assume template names reflect HMI types, so I would propose next changes:
```suggestion
      template = "MEDIA"
```

Reply:
@aderiabin Done in b2655a1d4abd36817f0f99596d92e656817ac74d